### PR TITLE
Expim async openai

### PR DESCRIPTION
**Pull request by MatanFriedmann**
Adding support for multiple generation using async for openai, maintaining simple single generation for num_sample=1

### CODE REVIEW

File: config/config_diff/config_images.yml [hunk 1]
```diff
@@ -22,13 +25,18 @@ meta_prompts:
...
     llm:
-        type: 'google'
-        name: 'gemini-1.5-pro-latest'
+        type: 'OpenAI'
+        name: 'gpt-4o'
+#        type: 'google'
+#        name: 'gemini-1.5-pro-latest'
     num_workers: 1
-    source_image_folder: ''
+    source_image_folder: '~/Desktop/source_images'
```
Comment (Eladlev) [inline config/config_diff/config_images.yml:33]: Please remove this path

File: utils/config.py [hunk 2]
```diff
@@ -86,18 +87,46 @@ def get_t2i_model(config: dict):
...
+        def generate_images(prompt, num_images=1):
+            if num_images == 1:
+                img_urls = generate_single_image(prompt)
+            else:
+                prompts = [prompt]*num_images
+                img_urls = asyncio.run(run_image_generation_batch(prompts))
+            return img_urls
 
         return generate_images
+        # return run_image_generation_batch
```
Comment (Eladlev) [inline utils/config.py:129]: Remove this line

**Review by Eladlev — APPROVED** [review]

File: config/config_diff/config_images.yml (hunk 1 above)
Comment (MatanFriedmann) [reply config/config_diff/config_images.yml]: Removed

File: utils/config.py (hunk 2 above)
Comment (MatanFriedmann) [reply utils/config.py]: Removed

**Pull request merged**